test/api_integration/apis/saved_objects/bulk_create.js

```diff
@@ -68,7 +68,7 @@ export default function ({ getService }) {
                   type: 'dashboard',
                   id: 'a01b2f57-fcfd-4864-b735-09e28f0d815e',
                   updated_at: resp.body.saved_objects[1].updated_at,
-                  version: 'WzgsMV0=',
+                  version: resp.body.saved_objects[1].version,
```

Comment:
Some SO API integration tests already had a "relaxed" version check like I've changed this to.

The usage stats collection additions caused flaky behavior for those tests that have "strict" version checks (version could be slightly different depending upon whether the usage stats `incrementCounter` finished before the operation in question, I am guessing). At any rate it seemed prudent to change these integration test assertions so that slight version deviations don't cause flaky behavior.